File: TeamspeakBot/TeamspeakBot/ts3_sdk_3.0.4.4/examples/client_android/app/src/main/java/com/teamspeak/ts3sdkclient/connection/ConnectionParams.java
package com.teamspeak.ts3sdkclient.connection;

/**
 * TeamSpeak 3 sdk client sample
 *
 * Copyright (c) 2007-2017 TeamSpeak-Systems
 *
 * @author Anna
 * Creation date: 10.02.17
 *
 * Required params to establish a server connection
 */
public class ConnectionParams {
    private String mServerAddress = "";
    private int mPort;
    private String mNickname = "";
    private String mIdentity = "";

    public ConnectionParams(String serverAddress, int port, String nickname, String identity) {
        mServerAddress = serverAddress;
        mPort = port;
        mNickname = nickname;
        mIdentity = identity;
    }

    public String getServerAddress() {
        return mServerAddress;
    }

    public int getPort() {
        return mPort;
    }

    public String getNickname() {
        return mNickname;
    }

    public String getIdentity() {
        return mIdentity;
    }
}

File: TeamspeakBot/TeamspeakBot/ts3_sdk_3.0.4.4/examples/client_android_javasound/app/src/main/java/com/teamspeak/ts3sdkclient/connection/ConnectionParams.kt
package com.teamspeak.ts3sdkclient.connection

/**
 * TeamSpeak SDK client sample
 *
 * Copyright (c) 2007-2018 TeamSpeak Systems
 *
 * Required parameters to establish a server connection
 */
data class ConnectionParams(
        var identity: String,
        var address: String = "127.0.0.1",
        var port: Int = 9987,
        var nickname: String = "TeamSpeakUser",
        var serverPassword: String = "")
